src/libraries/System.Text.Json/src/System.Text.Json.csproj

```diff
@@ -395,14 +395,15 @@ The System.Text.Json library is built-in as part of the shared framework in .NET
   </ItemGroup>
 
   <!-- Application tfms (.NETCoreApp, .NETFramework) need to use the same or higher version of .NETStandard's dependencies. -->
```

Comment:
Interesting. Isn't this statement true anymore? We had these explicit package refs because the package also provides a NS2.0 asset and its version could be higher / newer, ie in servicing.

Reply:
It's still true.  We do reference the latest version on current - that comes from the framework itself.

Reply:
What if the 9.0.1 package is used with  a 9.0 shared framework?

Reply:
We pin the assembly version for the major release so it's still sufficient.  We specifically want the behavior where we don't bring the serviced package and instead let it be handled by servicing the framework itself (that was the request in the issue).

Reply:
I wanted this behavior for a long time so I'm ok with this but I'm just surprised. Back, between NET 5 and NET 6 when Anirudh made these changes, we concluded / agreed on the current behavior.

Reply:
I think we had a bias towards servicing via packages and letting the library guarantee they'd be serviced.

We also hadn't fully formalized assembly pinning for everything in the shared framework, that came around the same time / slightly later.  For a while we were still bumping things.  That all had to stop in 6.0 when we decided to service ref-packs too.

Lately folks prefer to minimize the servicing responsibilities or libraries - so I think we can change this.

I'm also happy with this change.  I wish we could do it more - on older LTS frameworks - but we can't and still have live-package references from netstandard2.0 target frameworks.